### PR TITLE
Update release notes for v6.1

### PR DESCRIPTION
**Pull request by RichardWaiteSTFC**
**Description of work.**
Aims:

- [x] To remove duplicate entries
- [x] Add in hyperlinks where necessary
- [x] Spelling/grammar check
- [x] Remove some instances of "Fixed a bug" so the bug fixes sections don't get too repetitive!
- [x] Add image to index (collage)

**To test:**
(1) Build without error
(2) Try navigating links in html

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.1.0/index.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@
...
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../images/ReleaseNotesIndex_v6_1.png
    :class: screenshot
-   :width: 385px
+   :width: 440px
    :align: right
 
 .. contents:: Table of Contents
```
Comment (DanielMurphy22) [inline docs/source/release/v6.1.0/index.rst:12]: Remove the warning on line 15!

File: docs/source/release/v6.1.0/indirect_geometry.rst [hunk 2]
```diff
@@ -23,13 +23,13 @@ Improvements
...
+- The x range markers on the ISISDiagnostics plot of Data Reduction remain present.
 - Fixed a crash on the Data Analysis interface when attempting to drag the Start and End X sliders on the preview plot.
 - In IsoRotDiff, DiffSphere, and DiffRotDiscreteCircle Aliases have been removed to avoid clashes with interfaces.
-- Fixed a bug that caused an error warning when adding EISF data to F(q) fit if Width has already been added.
+- Stopped error warning thrown when adding EISF data to F(q) fit if Width has already been added.
 - Fixed a bug that caused the spectra list in F(q) fit to be blank when reopening the add workspace dialog.
-- Fixed a bug that erased previous fits in the Indirect Data Analysis fit tabs when changing to the full function view.
-- Fixed a bug that prevented fit ranges in Indirect Data Analysis from being negative.
+- Previous fits in the Indirect Data Analysis fit tabs no longer erased when changing to the full function view.
+- Fit ranges in Indirect Data Analysis can now be negative.
```
Comment (DanielMurphy22) [inline docs/source/release/v6.1.0/indirect_geometry.rst:33]: I love it! Bug fixes can be discussed positively and this is the way to do it!

File: docs/source/release/v6.1.0/diffraction.rst [hunk 3]
```diff
@@ -98,8 +98,7 @@ New features
 - Modified some logs in output workspace from :ref:`LoadWANDSCD <algm-LoadWANDSCD>` to be TimeSeriesProperty so they work with :ref:`SetGoniometer <algm-SetGoniometer>`.
 - :ref:`IntegratePeaksMD <algm-IntegratePeaksMD>` has option to integrate ellipsoids around estimated centroid instead of nominal position.
 - :ref:`IntegratePeaksMD <algm-IntegratePeaksMD>` has option to determine ellipsoid covariance iteratively and to use the estimated standard deviation rather than scale the major axis of the ellipsoid to the spherical radius.
-- :ref:`ConvertHFIRSCDtoMDE <algm-ConvertHFIRSCDtoMDE>` has new geometrical correction factor `ObliquityParallaxCoefficient` for shift in vertical beam position due to wide beam.
-- :ref:`ConvertWANDSCDtoQ <algm-ConvertWANDSCDtoQ>` has new geometrical correction factor `ObliquityParallaxCoefficient` for shift in vertical beam position due to wide beam.
+- Algorithms :ref:`ConvertWANDSCDtoQ <algm-ConvertWANDSCDtoQ>` and :ref:`ConvertHFIRSCDtoMDE <algm-ConvertHFIRSCDtoMDE>` have a new geometrical correction factor `ObliquityParallaxCoefficient` for shift in vertical beam position due to wide beam.
```
Comment (DanielMurphy22) [inline docs/source/release/v6.1.0/diffraction.rst:101]: Really nice to combine these!

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]
I **really** like the release image! These are probably the most vibrant and positive release notes yet! Only request is to remove the warning on the main page!

**Review by martyngigg — CHANGES_REQUESTED** [review]
This is great. I think our release page generator has a bug. This line at the bottom of `index.rst` is not quite right:

https://github.com/mantidproject/mantid/pull/31606/files#diff-9dc183af52e2489cf0328473ad7b885d7950c13384be8982c6068b810014c051R92

It should be 

```
.. _GitHub milestone: https://github.com/mantidproject/mantid/pulls?utf8=%E2%9C%93&q=is%3Apr+milestone%3A"Release 6.1"+is%3Amerged
```

**Comment by RichardWaiteSTFC** [comment]
> This is great. I think our release page generator has a bug. This line at the bottom of `index.rst` is not quite right:
> 
> https://github.com/mantidproject/mantid/pull/31606/files#diff-9dc183af52e2489cf0328473ad7b885d7950c13384be8982c6068b810014c051R92
> 
> It should be
> 
> ```
> .. _GitHub milestone: https://github.com/mantidproject/mantid/pulls?utf8=%E2%9C%93&q=is%3Apr+milestone%3A"Release 6.1"+is%3Amerged
> ```

Thanks for this - I tried replacing it with the link below, built and clicked the milestone hyperlink on the html - it directs to gituhub but it didn't put a space between Release and 6.1 - i.e. it filtered github PRs with
`is:pr milestone:"Release6.1" is:merged`
rather than (which does work)
`is:pr milestone:"Release 6.1" is:merged `
even though there is a space...

**Comment by martyngigg** [comment]
> > This is great. I think our release page generator has a bug. This line at the bottom of `index.rst` is not quite right:
> > https://github.com/mantidproject/mantid/pull/31606/files#diff-9dc183af52e2489cf0328473ad7b885d7950c13384be8982c6068b810014c051R92
> > It should be
> > ```
> > .. _GitHub milestone: https://github.com/mantidproject/mantid/pulls?utf8=%E2%9C%93&q=is%3Apr+milestone%3A"Release 6.1"+is%3Amerged
> > ```
> 
> Thanks for this - I tried replacing it with the link below, built and clicked the milestone hyperlink on the html - it directs to github but it didn't put a space between Release and 6.1 - i.e. it filtered PRs with
> `is:pr milestone:"Release6.1" is:merged`
> rather than (which does work)
> `is:pr milestone:"Release 6.1" is:merged `
> even though there is a space in the original link...

Oh yes, of course. It's a url so it can't contain a space. Try:
```
.. _GitHub milestone: https://github.com/mantidproject/mantid/pulls?utf8=%E2%9C%93&q=is%3Apr+milestone%3A"Release+6.1"+is%3Amerged
```

**Review by martyngigg — APPROVED** [review]
These look good to me. @DanielMurphy22 Once your approval is there I'll merge this.

**Review by DanielMurphy22 — APPROVED** [review]
Good job!

**Pull request merged**